src/stepper_interface.cpp

```diff
@@ -18,28 +35,25 @@ uint8_t MotorController::StepperInterface::getStepperWires() const {
 }
 
 void MotorController::StepperInterface::setStepperWires(const uint8_t newAlmount) {
-    if (newAlmount >= 3 && newAlmount <= 5) {
+    if (newAlmount == 2) {
+        stepperWires = newAlmount;
+    } else if (newAlmount == 4) {
+        stepperWires = newAlmount;
+    } else if (newAlmount == 5) {
         stepperWires = newAlmount;
     }
 }
 
-double MotorController::StepperInterface::getStride() const {
-    return stride;
+int16_t MotorController::StepperInterface::getSteps() const {

```

Comment:
Since setSteps can only take values higher than 4, a uint seems more in place here than an int